pyblish_aftereffects/util.py

```diff
@@ -0,0 +1,48 @@
+import os
+import socket
+
+
+def send(cmd, port=None):
+
+    host = '127.0.0.1'
+    if not port:
+        port = int(os.environ["PYBLISH_AFTEREFFECTS_PORT"])
+
+    # we expect a result no matter if it errors, so we keep trying until we
+    # get a reply. This is slow, but relyable.
+    keep_trying = True
+    result = ""
+    while(keep_trying):
```

Comment:
This could result in an infinite loop if for whatever reason the server never replies.

I would introduce a retry count.

```python
retries = 0
while retries < 3:
  try:
    ...
  except:
    retries += 1
```